package.json

```diff
@@ -35,7 +35,7 @@
     "lint:report": "mkdir -p build2 && run-p -c lint:eslint:report lint:grunt:report lint:stylelint:report",
     "test:dev": "grunt assets && karma start",
     "test:coverage": "yarn test --reporters=progress,coverage",
-    "test": "yarn test:dev --single-run",
+    "test": "yarn test:dev --single-run && yarn jest",
```

Comment:
Would prefer we separate Jest from our existing test suite, as we're now running Jest **on top** of our existing test suites. This adds extra test time that isn't necessary.

To start - I think splitting them into a separate `test:jest` command (and setup the necessary Bacon script) would be ideal. This allows developers to run jest tests independently.

Reply:
`yarn jest` will run all the jest tests. We can add an alias `test:jest` if you think that is useful

Reply:
I know I tend to look in package.json for how to run things - far too often the configs are tucked away so I can't expect the simple case to work.